src/main/java/teammates/common/datatransfer/attributes/StudentAttributes.java

```diff
@@ -372,4 +372,14 @@ public boolean isSectionChanged(StudentAttributes originalStudentAttribute) {
     public boolean isTeamChanged(StudentAttributes originalStudentAttribute) {
         return this.team != null && !this.team.equals(originalStudentAttribute.team);
     }
+    
+    /**
+     * Checks whether the edit form of student has changed the email value.
+     * 
+     * @param originalStudentAttribute
+     * @return true if email value has changed from its original value.
```

Comment:
This line can be removed and the overview sentence can be changed to `Returns true if email value has changed from its original value`